content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -571,229 +568,148 @@ Push the schema with a custom schema location:
+| :------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
+| `--schema`    | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
 
-##### Preview the migration that would be applied by running the `migrate up` command
+#### Examples
 
 ```terminal
-prisma migrate up --preview --experimental
+prisma migrate deploy --preview-feature
 ```
 
-### <inlinecode>migrate down</inlinecode>
-
-Migrate the database down to a specific state. 
-
-#### Prerequisites
-
-Before using the `migrate down` command, you must define a valid [`datasource`](../../concepts/components/prisma-schema/data-sources) within your `schema.prisma` file.
+### <inlinecode>migrate resolve</inlinecode>
 
-For example, the following `datasource` defines a SQLite database file within the current directory:
-
-```prisma
-datasource db {
-  provider = "sqlite"
-  url      = "file:my-database.db"
-}
-```
+The `migrate resolve` command allows you to solve migration history issues in production by marking a migration as already applied (supports baselining) or rolled back.
 
 #### Arguments
 
-The point to migrate back to can be defined in any of the following three ways:
-
-| Argument  | Required | Description                                                                             | Default |
-| --------- | -------- | --------------------------------------------------------------------------------------- | ------- |
-| decrement | No       | Specifies the number of backwards migrations to apply.                                  | 1       |
-| name      | No       | Specifies where to migrate back to using the name of the final migration to apply.      |
-| timestamp | No       | Specifies where to migrate back to using the timestamp of the final migration to apply. |
-
-#### Options
-
-Additionally, the following options modify the behavior of the `migrate down` command:
+| Argument       | Required | Description                                                                                                                                       | Default                                          |
+| :------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
+| `--applied` | No*       | Record a specific migration as applied - for example `--applied "20201231000000_add_users_table"` |  |
+| `--rolledback` | No*       | Record a specific migration as rolled back - for example `--rolledback "20201231000000_add_users_table"`. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
```

Comment:
```suggestion
| `--rolled-back` | No*       | Record a specific migration as rolled back - for example `--rolled-back "20201231000000_add_users_table"`. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
```
Also could you check everywhere that it's `--rolled-back` and not `--rolledback`?

Reply:
Checked and fixed